src/widgets/dialogs/EmotePopup.cpp

```diff
@@ -156,6 +156,23 @@ EmotePopup::EmotePopup(QWidget *parent)
 
     this->loadEmojis();
 
+    // CTRL + 1-8 to open corresponding tab
+    for (auto i = 0; i < 8; i++)
+    {
+        char hotkey[7];
+        std::sprintf(hotkey, "CTRL+%d", i + 1);
+        const auto openTab = [this, i, notebook] {
+            notebook->selectIndex(i);
+        };
+        createWindowShortcut(this, hotkey, openTab);
```

Comment:
```suggestion
        const auto openTab = [this, i, notebook] {
            notebook->selectIndex(i);
        };
        createWindowShortcut(this, QString("CTRL+%1").arg(i + 1), openTab);
```

Please do the same thing wherever you copied this from (widgets/Window.cpp)